<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Addvariables</class>
 <widget class="QDialog" name="Addvariables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>225</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>225</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add</string>
  </property>
  <widget class="QLabel" name="lName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tValue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bOK">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bCancel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
